src/index.html

```diff
@@ -12,6 +12,178 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="item">
+          <label for = "surname">
+                Surname:
+          </label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            minlength="2"
+            maxlength="15"
+            required
+          >
+        </div>
+        <div class="item">
+          <label for = "name">
+                Name:
+          </label>
+          <input
+            type = "text"
+            id = "name"
+            name = "name"
```

Comment:
```suggestion
            type="text"
            id="name"
            name="name"
```
Пробелы можно не оставлять